cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: apps/v1beta1
+kind: StatefulSet
```

Comment:
Why not deployment? Replica=1 and fixed pvc sounds like deployment is the right choice.